datahub-web-react/src/app/homeV3/toast/useShowToast.tsx

```diff
@@ -0,0 +1,37 @@
+import '@app/homeV3/toast/notification-toast-styles.less';
+
+import { Icon, Text, colors } from '@components';
+import { notification } from 'antd';
+import React from 'react';
+
+const notificationStyles = {
+    backgroundColor: colors.blue[0],
+    borderRadius: 8,
+    width: 475,
```

Comment:
is it possible to not set a width or to let the content determine the width? just thinking for future use of this nice reusable hook